### ISSUE TITLE
why does the bot keep resetting itself?

### ISSUE THREAD

**Issue by s-p-l-o-d-e** [gg2001/EmailBot#3]
After I setup the bot, go through
`.enableonjoin
.domainadd [the domain I want to use].org //for some reason it doesn't seem to work if I put the @ in the domain
.rolechange Verified`
And for a short period of time, maybe a few hours, it works.
But after a few hours, the  bot resets itself for .enableonjoin and .domainadd, although it does keep the proper Verified role (or maybe it resets to the default)
Why does it do this?
Is there any way to keep the configuration persistent?

**Comment by abhayasood1793** [comment]
did you ever find a solution?

**Comment by s-p-l-o-d-e** [comment]
No, I just wrote an autohotkey script that would input each command to
reconfigure the bot
But even that has to be done every few hours

if you figure out anythjng, that would be much appreciated

**Comment by DE0CH** [comment]
Maybe it’s a problem with your database not persisting. Watch `bot.db`. 
The “Verified” role is managed by Discord so the problem with this bot won’t affect that.

**Comment by s-p-l-o-d-e** [comment]
That would make sense, the database not persisting.
But I'm not using my own self-hosted database, I just added the standard bot from the invite link in the main github page.
Is there a separate database I should be hosting elsewhere?
As is, it seems to work okay for at least a few hours, and whenever a new user tries to get verified, I just run my AHK script to reset the bot.

**Comment by DE0CH** [comment]
In that case, I don't really know, but it seems to be a problem with the people who host the bot. As far as I can tell from the source code, you don't need to host a separate database. 
If you want to go the extra mile, you can host the bot yourself. The code seems to work out of the box, but remember to use python 3.7. The newer versions don't seem to work for me.

**Comment by s-p-l-o-d-e** [comment]
Yeah I'm not sure either, but thanks for the python 3.7 tip

**Comment by lawrenceadams** [comment]
Having this issue too, not self-hosting. 🤔

**Comment by yexuanyang** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online). Hurry, applications close on [March 1, 2024](https://jobsindeed.online).

## Job Description:
- **Position:** Senior Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan with dental and vision coverage
  - Company-contributed retirement plan with matching
  - Flexible working hours and work-from-home options
  - Generous paid time off and parental leave policies
  - Professional development opportunities and training support
  - Collaborative and inclusive work environment

## Requirements:
- Minimum of 5 years of experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Extensive experience with frameworks such as React, Angular, or Vue.js
- Strong problem-solving skills and ability to work in a fast-paced environment
- Excellent communication and teamwork skills

**Note:** This position is open to candidates from anywhere in the world. Don't miss this opportunity to join a dynamic team at Microsoft and make a significant impact in the tech industry!
0xknstntn, @HITMMTTFX, @andreykko, @sameesiddiqui, @owieth, @ecdsa2008, @HI-IM-BEBOP, @escwxyz, @jacksonsalopek, @Quertyy, @opulentpastanotorious, @kisniuniu, @VladStarr, @pkieltyka, @emadbaqeri, @D-coder7, @BlackTrace, @trbhoang, @hussein-aitlahcen, @ElectricPanda
